README.md

```diff
@@ -228,6 +228,43 @@ try {
 ```
 Note that in the closure-based example above, exceptions are intercepted automatically.
 
+### Oversampling Mitigation
+Oversampling mitigation allows you to ignore a parent segment/subsegment's sampled flag and instead set it to false.
+The code below demonstrates overriding the sampled flag based on the SQS messages sent to Lambda.
+
+```java
+public class Handler implements RequestHandler<SQSEvent, String> {
+  public Handler() {
+  }
+
+  @Override
+  public String handleRequest(SQSEvent event, Context context) {
+
+    // Check to see if any messages upstream are sampled.
+    boolean toSample = false;
+    for (SQSMessage message: event.getRecords()) {
+      if (SQSMessageHelper.isSampled(message)) {
+        toSample = true;
```

Comment:
Wouldn't this always sample the subsegment even when a single message in the batch is sampled?
I thought how oversampling mitigation would work is that when processing each message in the batch, we decided whether to begin a subsegment with/without sampling based on if the message is sampled or not.

Reply:
I wanted to demonstrate how to do a fan-in.